tmt/steps/execute/scripts/tmt-reboot

```diff
@@ -1,12 +1,52 @@
 #!/bin/bash
+PATH=/sbin:/usr/sbin:$PATH
+
+create_nextboot_timer() {
+        if [ -z "$NEXTBOOT_VALID_TIME" ] ; then
+                NEXTBOOT_VALID_TIME=180
+        fi
+
+        # For future reference, it's possible to use `at` if supported everywhere
+        nohup bash -c "
+                logger -s 'NextBoot value is valid for the next $NEXTBOOT_VALID_TIME seconds';
+                sleep $NEXTBOOT_VALID_TIME;
+                logger -s 'Purging NextBoot value...'
+                efibootmgr -N && logger -s 'NextBoot value is no longer valid'
+        " 2>&1 | /usr/bin/tee /dev/console &
+}
+
 command=""
 timeout=""
-while getopts "c:t:" flag; do
+efi=True
+while getopts "c:t:e" flag; do
     case "${flag}" in
         c) command="${OPTARG}";;
         t) timeout="${OPTARG}";;
+        e) efi=False;;
     esac
 done
+
+if [ $efi ]; then
+    if efibootmgr &>/dev/null ; then
+        os_boot_entry=$(efibootmgr | awk '/BootCurrent/ { print $2 }')
+        # fall back to /root/EFI_BOOT_ENTRY.TXT if it exists and BootCurrent is not available
+        if [[ -z "$os_boot_entry" && -f /root/EFI_BOOT_ENTRY.TXT ]] ; then
+            os_boot_entry=$(</root/EFI_BOOT_ENTRY.TXT)
+        fi
+        if [[ -n "$os_boot_entry" ]] ; then
+            logger -s "efibootmgr -n $os_boot_entry"
+            efibootmgr -n $os_boot_entry
+            create_nextboot_timer
+            # Adjust watchdog if running inside of test case
+            if [[ -n $RSTRNT_RECIPE_URL && $RSTRNT_MAXTIME ]] ; then
+                rstrnt-adjust-watchdog $(($RSTRNT_MAXTIME + $NEXTBOOT_VALID_TIME))
```

Comment:
I guess this part can be removed as it is related to `restraint` only and `tmt` does not provide `rstrnt-adjust-watchdog`.